docs/spec.md

```diff
@@ -1979,7 +1992,8 @@ If `feature_group_count = 1` and `batch_group_count = 1`, then for all
 `output_spatial_index` in the index space of `dim(result, output_spatial_dimensions)`,
 `result[result_shape(:, output_spatial_index, :)] = dot_product` where:
 
-* `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations[:] - 1)`.
+* `padded_lhs_value = is_quantized_tensor(lhs) ? zero_point(lhs) : 0`.
```

Comment:
Also, what is the element type of `padding_value`? Let's express that explicitly by saying `padding_value = constant(is_quantized_tensor(lhs) ? zero_point(lhs) : 0, ???)`.

I see there was a conversation about this in https://github.com/openxla/stablehlo/pull/1477#discussion_r1194496076, and the conclusion was that "This 0 value is expected to be interpreted as a value of type equal to the lhs, which could be integer, floating-point, or complex type", but I think we should make this explicit to 100% avoid confusion.

Reply:
done. PTAL.